### PR TITLE
Fix target="_blank" space & code indentation

### PR DESCRIPTION
**Pull request by williankeller**
## Purpose of this pull request

- Improve Varnish code example & indentation.
- Fix `{:target="_blank"}` space at the _Warning block area (RFC-2616 section 13)_.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/extension-dev-guide/cache/page-caching/public-content.html
- https://devdocs.magento.com/guides/v2.2/extension-dev-guide/cache/page-caching/public-content.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cache/page-caching/public-content.html

## Links to Magento source code
- https://github.com/magento/magento2/blob/2.0/app/code/Magento/PageCache/etc/varnish4.vcl#L63-L68

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md [hunk 1]
```diff
@@ -125,18 +125,18 @@ class CustomerAgeContextPlugin
 
 The `subject->setValue` argument specifies the value for newcomer context and is used to guarantee parity during cache key generation for newcomers and users who already received the `X-Magento-Vary` cookie.
 
-For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
+For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2300url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
```
Comment (erikmarr) [inline guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]:
```suggestion
For another example of a context class, see [Magento/Framework/App/Http/Context]({{ site.mage2100url }}lib/internal/Magento/Framework/App/Http/Context.php){:target="_blank"}.
```

File: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md [hunk 2]
```diff
@@ -125,18 +125,18 @@ class CustomerAgeContextPlugin
...
 
 ### `X-Magento-Vary` cookie
 
-Use the `X-Magento-Vary` cookie to transfer context on the HTTP layer. HTTP proxies can be configured to calculate a unique identifier for cache based on the cookie and URL. For example, [our sample Varnish 4 configuration]({{ site.mage2000url }}app/code/Magento/PageCache/etc/varnish4.vcl#L63-L68){:target="_blank"} uses the following:
+Use the `X-Magento-Vary` cookie to transfer context on the HTTP layer. HTTP proxies can be configured to calculate a unique identifier for cache based on the cookie and URL. For example, [our sample Varnish 4 configuration]({{ site.mage2300url }}app/code/Magento/PageCache/etc/varnish4.vcl#L113-L130){:target="_blank"} uses the following:
```
Comment (erikmarr) [inline guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]:
```suggestion
Use the `X-Magento-Vary` cookie to transfer context on the HTTP layer. HTTP proxies can be configured to calculate a unique identifier for cache based on the cookie and URL. For example, [our sample Varnish 4 configuration]({{ site.mage2100url }}app/code/Magento/PageCache/etc/varnish4.vcl#L113-L130){:target="_blank"} uses the following:
```
is the varnish update specific to 2.3 here?

File: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ You can clear cached content immediately after a entity changes. Magento uses  `
 
 This section shows you how to tell Magento what cache to clear when you change an entity.
 
-First, your entity {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} must implement [`Magento/Framework/DataObject/IdentityInterface`]({{ site.mage2000url }}lib/internal/Magento/Framework/DataObject/IdentityInterface.php){:target="_blank"} as follows:
+First, your entity {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} must implement [`Magento/Framework/DataObject/IdentityInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/DataObject/IdentityInterface.php){:target="_blank"} as follows:
```
Comment (erikmarr) [inline guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]:
```suggestion
First, your entity {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} must implement [`Magento/Framework/DataObject/IdentityInterface`]({{ site.mage2100url }}lib/internal/Magento/Framework/DataObject/IdentityInterface.php){:target="_blank"} as follows:
```

**Comment by erikmarr** [comment]
@williankeller thanks for your feedback! This is unfortunately a known issue that we hope to resolve in a future release when we incorporate a more robust branching strategy. I've reviewed your changes and provided specific comments/suggestions.

File: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md (hunk 1 above)
Comment (williankeller) [reply guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]: Hi @erikmarr,
So the is the change the variable to mage2100url instead? Then the 2.3 pages will get the 2.1 repository still, right?

Comment (erikmarr) [reply guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]: Yeah linking to 2.0 needs to be fixed and while 2.1 is not ideal either, it's the compromise for now.

**Comment by lorikrell** [comment]
To allow for a specific version per version, we can remove the symlink files. Then create new pages per version, with just the changed link to the code repos per file.

**Comment by williankeller** [comment]
Thanks, guys!
@lorikrell, so, just to be clear, should I create new files instead of keeping the links, right? That's what I thought in the begin _(my point of view the right approach)_. Or should I change the value under 2100 as @erikmarr suggested?

**Comment by jeff-matthews** [comment]
@williankeller,

The Magento Documentation team needs to discuss the implications of breaking symlinks for this purpose. Please standby for more information.

See my comments in https://github.com/magento/devdocs/pull/4496 and https://github.com/magento/devdocs/issues/4422.

**Comment by erikmarr** [comment]
Hello @williankeller thanks again for you feedback. We were able to resolve the github links internally through the following PR:
https://github.com/magento/devdocs/pull/4518
can you update your PR to remove those changes? I believe that would leave just the  {:target="_blank"} issue?

**Comment by williankeller** [comment]
Hi @erikmarr, sure! I saw the provided fix about the blob variable, nice!
Sure I will change it to keep only the necessary changes.
Thanks!

**Comment by williankeller** [comment]
Hi @erikmarr 
I've applied the changes as requested. Could you please revalidate it?

**Pull request merged**